pkg/mapper/conditions_changed_test.go

```diff
@@ -0,0 +1,94 @@
+package mapper
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestIfConditionsChangedMapper(t *testing.T) {
```

Comment:
should we also have a scenario for multiple conditions, for example one where there is a change in the ignored fields and but also in one of the not ignored fields ( like the `Status` ) ?

Reply:
Good point. I redid the impl as a predicate as suggested by @MatousJobanek so the tests are not completely different.